hw/ip/prim/prim_present.core

```diff
@@ -0,0 +1,40 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+name: "lowrisc:prim:present"
+description: "PRINCE and PRESENT block cipher implementation"
+filesets:
+  files_rtl:
+    depend:
+      - lowrisc:prim:assert
+      - lowrisc:prim:cipher_pkg:0.1
+    files:
+      - rtl/prim_subst_perm.sv
+      - rtl/prim_present.sv
```

Comment:
hmm... this is a little strange. How come `prince` and `present` are all in the `present` core file?
Would it be better if we renamed this `prim_lw_cipher`? Then it can include all the ones in that category. 

lw = light-weight.

Reply:
Hmm, good point. I'll change the name.

Reply:
if we're calling this `prim_lw_cipher`, should the package also be called like that?
or vice versa, we could just call it `prim_cipher.core` (probably less edits).
sorry for the name bikeshedding ;):

Reply:
I don't have strong feelings. Let's go for `prim_cipher.core` and we can always rename them together later if we decide to.